src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,31 @@
+package core.basesyntax;
+
+public class Circle extends Shape {
+    private final double radius;
+
+    public Circle(double radius, String color) {
+        super(color);
+        this.radius = radius;
+    }
+
+    @Override
+    public double getArea() {
+        return Math.PI * radius * radius;
+    }
+
+    @Override
+    public String toString() {
+        return "Circle{"
+                + "radius="
+                + radius + ", color='"
+                + getColor() + '\''
+                + '}';
+    }
```

Comment:
From requirements: "you shouldn't override toString() method for this". Let's completely remove it here and in all other figures. You already have the method `draw()`.